fedora-pr-review-guide.md

```diff
@@ -74,6 +76,17 @@ Copy & paste to your PR review.
     - Set it to ASSIGNED (and *take* it) when you start working on it
     - Put the link to the PR in the BZ once it exists (even WIP), set the status to POST
 
+### Patches
+
+- If your PR adds new patches (or significantly reworks some patches):
+  - Properly document the purpose of the patch in the specfile.
```

Comment:
What does `properly` mean? I'd prefer to be specific here, so that we don't add confusion. If this is just a "strongifier", I'd remove the word and leave just `Document the purpose of the patch in the specfile.`

Reply:
Added in fixup.